packages/xo-web/src/common/editable/index.js

```diff
@@ -343,68 +336,61 @@ export class Number extends Component {
 
 export class Select extends Editable {
   static propTypes = {
-    options: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
-      .isRequired,
     renderer: PropTypes.func,
   }
 
-  componentWillReceiveProps (props) {
-    if (
-      props.value !== this.props.value ||
-      props.options !== this.props.options
-    ) {
-      this.setState({
-        valueKey: findKey(props.options, option => option === props.value),
-      })
-    }
-  }
-
-  get value () {
-    return this.props.options[this.state.valueKey]
+  _closeEditionIfUnfocused = () => {
+    this._focused = false
+    setTimeout(() => {
+      !this._focused && this._closeEdition()
+    }, 10)
   }
 
-  _onChange = event => {
-    this.setState({ valueKey: getEventValue(event) }, this._save)
+  _focus = () => {
+    this._focused = true
   }
 
-  _optionToJsx = (option, key) => {
-    const { renderer } = this.props
-
+  _renderDisplay () {
+    const { children, renderer, value } = this.props
     return (
-      <option key={key} value={key}>
-        {renderer ? renderer(option) : option}
-      </option>
+      children || <span>{renderer ? renderer(value) : this._renderer()}</span>
     )
   }
 
-  _onEditionMount = ref => {
-    // Seems to work in Google Chrome (not in Firefox)
-    ref && ref.dispatchEvent(new window.MouseEvent('mousedown'))
-  }
+  _renderer = () => {
+    const { value, multi } = this.props
+    if (value === undefined || (multi !== undefined && isEmpty(value))) {
+      return _('noValue')
+    }
 
-  _renderDisplay () {
-    const { children, renderer, value } = this.props
+    return multi === undefined ? value : value.map(v => `${v.label} `)
```

Comment:
`value.map(v => v.label).join(', ')`
(clearer separator and you don't want it after the last one)